### ISSUE TITLE
[Suggestion] User mode and developer mode (with more control)

### ISSUE THREAD

**Issue by Koliham** [withcatai/catai#26]
Hi,
it would be good to have some kind of user mode and developer mode, which can be toggled with an environment variable.
So you have more parameters to choose from in developer mode and when you are ready, you ship it in user mode with a simple interface.

**Comment by ido-pluto** [comment]
Actually, I created something like that, currently, this is in the preview. You can install this version like that:
```
npm i -g catai@1
```
I did not release that because I need more feedback, I am unsure what bug there is and what more would be good.
If you have feedback, it will be much appreciated :)

**Comment by Koliham** [comment]
Got an error installing it with "npm i -g catai@1"

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'catai@1.0.1',
npm WARN EBADENGINE   required: { node: '>=19' },
npm WARN EBADENGINE   current: { node: 'v18.16.0', npm: '9.5.1' }
npm WARN EBADENGINE }
npm ERR! code 1
npm ERR! path /usr/local/lib/node_modules/catai
npm ERR! command failed
npm ERR! command sh -c node scripts/migration/index.js
npm ERR! node:internal/process/promises:288
npm ERR!             triggerUncaughtException(err, true /* fromPromise */);
npm ERR!             ^
npm ERR!
npm ERR! [Error: ENOENT: no such file or directory, scandir '/root/catai/models'] {
npm ERR!   errno: -2,
npm ERR!   code: 'ENOENT',
npm ERR!   syscall: 'scandir',
npm ERR!   path: '/root/catai/models'
npm ERR! }
npm ERR!
npm ERR! Node.js v18.16.0

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-06-21T19_05_01_337Z-debug-0.log
```

My current models directory is in `/home/myusername/catai/models`
Did version 1 change the models path?

**Comment by ido-pluto** [comment]
No, but it migrates changes from version 0 to 1. 

Try to update nodejs to version 20, and try again.

Did you run this with sudo? That weird he thinks your home directory is root

**Comment by Koliham** [comment]
I had permission issues with some folders:
```
npm ERR! Error: EACCES: permission denied, symlink '../lib/node_modules/catai/scripts/cli.js' -> '/usr/bin/catai'
npm ERR!  [Error: EACCES: permission denied, symlink '../lib/node_modules/catai/scripts/cli.js' -> '/usr/bin/catai']
```
But I was finally able to install it:
- Can you implement a "--hide-settings" function so it is possible to host the application for external users without giving them access to sensitive settings?
- Can you check the height of the dialog:
![grafik](https://github.com/ido-pluto/catai/assets/7567492/9c7f722f-9af8-4032-b859-f619c288afa8)

**Comment by ido-pluto** [comment]
You can set the environment variable 'CATAI_ADMIN_SETTINGS' to false.
I will check the dialog model, what is the height of your screen?

**Comment by ido-pluto** [comment]
Fixed, on version 1.0.2
